packages/api-augment/src/interfaces/automationTime/definitions.ts

```diff
@@ -3,14 +3,6 @@ import { runtime } from './runtime';
 
 const automationTime = {
   rpc: {
-    getTimeAutomationFees: {
-      description: 'Retrieve automation fees',
-      params: [
-        { name: 'action', type: 'AutomationAction' },
```

Comment:
Yeah, looking at this definititon. The AutomationAction is so not flexible comparing to the `params: [ { name: 'extrinsic', type: 'Extrinsic' } ]` in `queryFeeDetails`